endpoints/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_subdirectory(shared)
+
+add_subdirectory(authentication)
+add_subdirectory(collections)
+#add_subdirectory(config)
```

Comment:
I didn't see config anywhere before... Is this something that's planned?

Reply:
It's referenced in a couple commented out places here:
https://github.com/irods/irods_client_http_api/blob/38d67aa5ab798952450dc4429c20d82586a375e8/src/endpoints/CMakeLists.txt#L35
https://github.com/irods/irods_client_http_api/blob/38d67aa5ab798952450dc4429c20d82586a375e8/src/CMakeLists.txt#L56
And one place where it's not commented out:
https://github.com/irods/irods_client_http_api/blob/38d67aa5ab798952450dc4429c20d82586a375e8/src/endpoints/handlers.hpp#L19

Reply:
It is a placeholder for when we decide to allow modifying iRODS configuration remotely.